Description: Invoking shell functions with
 environment-variable=value shellfunction
 does not work in dash. Use export export/unset in testsuite instead.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Bug: https://sourceforge.net/p/autogen/bugs/164/
Forwarded: https://sourceforge.net/p/autogen/bugs/164/

--- autogen-5.18.4.orig/agen5/test/format.test
+++ autogen-5.18.4/agen5/test/format.test
@@ -79,13 +79,15 @@ document.write('" >Bruce Korb</a>');
 </script>
 _EOF_
 
-AUTOGEN_DNE_DATE=false \
+export AUTOGEN_DNE_DATE=false &&
     run_ag x ${testname}.def || failure autogen failed
+unset AUTOGEN_DNE_DATE
 cmp -s ${testname}.base ${testname}.test || \
    failure "`diff ${testname}.base ${testname}.test`"
 
-AUTOGEN_DNE_DATE=true \
+export AUTOGEN_DNE_DATE=true &&
     run_ag x ${testname}.def || failure autogen failed
+unset AUTOGEN_DNE_DATE
 dte=`sed -n 's/.*It has been AutoGen-ed *//p' ${testname}.test`
 test "X${dte}" = "X" && \
   failure "expected a date on the 'It has been AutoGen-ed' line${nl}`
